Libraries/Alert/Alert.js

```diff
@@ -154,10 +173,53 @@ class Alert {
       // [TODO(macOS GH#774)
     } else if (Platform.OS === 'macos') {
       const defaultInputs = [{default: defaultValue}];
-      AlertMacOS.prompt(title, message, callbackOrButtons, type, defaultInputs);
+      Alert.promptMacOS(title, message, callbackOrButtons, type, defaultInputs);
     }
     // ]TODO(macOS GH#774)
   }
+
+  // [TODO(macOS GH#774)
```

Comment:
i feel like old code had better comments on the params?

Reply:
You're right. I'll see if I can copy the documentation from AlertMacOS to promptMacOS